src/jit/emit.cpp

```diff
@@ -2953,6 +2953,36 @@ void                emitter::emitDispVarSet()
 
 /*****************************************************************************/
 #endif//DEBUG
+
+//------------------------------------------------------------------------  
+// emitSetSecondRetRegGCType: Sets the GC type of the second return register for instrDescCGCA struct.  
+//  
+// Arguments:  
+//    id            - The large call instr descriptor to set the second GC return register type on.  
+//    secondRetSize - The EA_SIZE for second return register type.  
+//  
+// Return Value:  
+//    None  
+//  
+
+void            emitter::emitSetSecondRetRegGCType(instrDescCGCA* id, emitAttr secondRetSize)
+{
+#if defined(FEATURE_MULTIREG_RET)
+    if (EA_IS_GCREF(secondRetSize))
+    {
+        id->idSecondGCref(GCT_GCREF);
+    }
+    else if (EA_IS_BYREF(secondRetSize))
+    {
+        id->idSecondGCref(GCT_BYREF);
+    }
+    else
+    {
+        id->idSecondGCref(GCT_NONE);
+    }
+#endif // defined(FEATURE_MULTIREG_RET)  
```

Comment:
I see all the calls to this method are under #if FEATURE_MULTIREG_RET, in such a case shouldn't both function decl/definition be under #if FEATURE_MULTIREG_RET?